README.md

```diff
@@ -1,87 +1,107 @@
-## Version 0.7.0
+<h1 align="center">OSTIS example app</h1>
 
-This version allows communication with the knowledge base via [JSON-based Websocket protocol](http://ostis-dev.github.io/sc-machine/http/websocket/).
+[![license](https://img.shields.io/badge/License-MIT-yellow.svg)](LICENSE)
 
-The new version of web interface communicates with **sc-server** using two-way JSON protocol.
+## About
 
-## Install
+OSTIS example app is an ostis-system based on [**ostis-web-platform**](https://github.com/ostis-ai/ostis-web-platform) and designed with [OSTIS Technology](https://github.com/ostis-ai).
 
-Linux:
+0.7.0 version allows communication with the knowledge base via [JSON-based Websocket protocol](http://ostis-dev.github.io/sc-machine/http/websocket/).
+
+
+## Installation
+
+For Ubuntu/Debian based distros:
 ```sh
 git clone https://github.com/ostis-apps/ostis-example-app.git
 cd ostis-example-app/scripts
-git checkout 0.7.0
 ./install_ostis.sh
 ```
 
 ## Build knowledge base
-Linux:
+
+Before first launch or after changes in KB you should build knowledge base. 
+
 ```sh
 cd ostis-example-app/scripts
 ./build_kb.sh
 ```
 
-## Run
+## Usage
 
-Run on Linux (both terminals or terminal tabs should work at the same time):
+To launch system you should start sc-server:
 ```sh
-#Terminal 1
 cd ostis-example-app/scripts
 ./run_sc_server.sh
+```
+
+After that launch sc-web interface:
 
-#Terminal 2
+```sh
 cd ostis-example-app/scripts
 ./run_scweb.sh
 ```
 
-Then open localhost:8000 in your browser.
+To check that everything is fine open localhost:8000 in your browser.
 ![](https://i.imgur.com/6SehI5s.png)
 
 ## Project Structure
 
-### kb
-The place for the knowledge base of your app. Put your **.scs** files here.
+### Knowledge Base
 
-### problem-solver
-The place for the problem solver of your app. Put your agents here.
+`kb` is the place for the knowledge base source text files of your app. Put your **.scs** and **.gwf** files here.
 
-*Use **scp_stable** branch for the development of agents on SCP.*  
-*Use **0.5.0**, **0.6.0** or **0.7.0** branch for the development of agents on C++.*  
+### Problem Solver
 
+`problem-solver` is the place for the problem solver of your app. Put your agents here. After changes in problem-solver you should rebuild it:
 
-#### Agents on C++
-Some tips:
-- Store your modules with C++ agents in *problem-solver/cxx*;
-- After updating your C++ code you need to rebuild problem-solver. Just run:  
-```
+*Use **scp_stable** branch for the development of agents on SCP.*  
+*Use **0.5.0**, **0.6.0** or **0.7.0** branch for the development of agents on C++. You can see an example module with a C++ agent [here](problem-solver/cxx/exampleModule/README.md)*  
```

Comment:
Should we mention here previous versions? If we should, can you mention that last version is much more stable?

Reply:
Which versions? scp_stable(0.4.0) and other are release versions and can be used. Last version is often more stable, not?

Reply:
Ok, got it